Orange/widgets/data/owsave.py

```diff
@@ -89,6 +90,10 @@ def __init__(self):
 
         box.layout().addLayout(form)
 
+        self.annotations_cb = gui.checkBox(
+            self.controlArea, self, "add_type_annotations", label="Add type annotations",
+        )
+
```

Comment:
You can fix the layout (checkbox not in the box, but outside and with a smaller indent) by changing this to:

```
        self.annotations_cb = gui.checkBox(
            None, self, "add_type_annotations", label="Add type annotations",
        )
        form.addRow(self.annotations_cb, None)
```

The layout of this widget is awful anyway,  but this is another issue.